Description: Use debian's help2man instead of the version distributed
Forwarded: not-needed
Origin: vendor
Author: Don Armstrong <don@debian.org>
--- a/stepmake/stepmake/help2man-rules.make
+++ b/stepmake/stepmake/help2man-rules.make
@@ -1,24 +1,8 @@
-# We must invoke the generated $(outdir)/help2man script instead of
-# the help2man.pl source, which means that the scripts/build directory
-# must be built first.
-#
-# From the perlrun man-page:
-#
-#	If the #! line does not contain the word "perl", the
-#       program named after the #! is executed instead of the Perl
-#       interpreter.  This is slightly bizarre, but it helps
-#
-# Indeed it is.  Perl sucks.
-#
-# Two screenfulls explaining that the otherwise standard #! is broken
-# for perl, and arguing that this broken magic is better in some
-# cases.  Four more explaining what a line comment is, and that it may
-# be parsed, same here.
-
+# The distributed help2man sucks; use the Debian version instead.
 ifeq ($(PYTHON),$(TARGET_PYTHON))
-HELP2MAN_COMMAND = $(buildscript-dir)/help2man $< > $@
+HELP2MAN_COMMAND = help2man $< > $@
 else
-HELP2MAN_COMMAND = $(buildscript-dir)/help2man "env $$(basename $$(head -1 $<)) $<" > $@
+HELP2MAN_COMMAND = help2man "env $$(basename $$(head -1 $<)) $<" > $@
 endif
 
 ifeq ($(strip $(CROSS)),no)
@@ -38,6 +22,3 @@
 $(outdir)/%.1: out/%.1
 	cp $< $@
 endif
-
-$(buildscript-dir)/help2man:
-	$(MAKE) -C $(depth)/scripts/build
